python/nano/tutorial/inference/pytorch/pytorch_save_and_load_onnx.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Required Dependecies
+
+# ```bash
+# pip install onnx onnxruntime
+# ```
+
+import torch
+from torchvision.models import resnet18
+
+if __name__ == "__main__":
+    model_ft = resnet18(pretrained=True)
+
+    x = torch.rand(2, 3, 224, 224)
+    model_ft.eval()
```

Comment:
no need to switch to eval mode

Reply:
At present， it seems that we don't set `model.eval()` in openvino and onnxruntime.

Reply:
> At present, it seems that we don't set `model.eval()` in openvino and onnxruntime.

So you think maybe we could add `model.eval` here?

Reply:
Yeah, i think could add here.  Maybe we should add `model.eval()` in `trace` and `quantize` later.